### ISSUE TITLE
Copay Failing: BadRequest

### ISSUE THREAD

**Issue by KeyJockey** [bitpay/copay#4981]
Hi Guys:

Copay totally failing this morning... cannot send any transaction, error message coming up: BADREQUEST
![badrequest](https://cloud.githubusercontent.com/assets/13903392/20007505/ede96ab6-a272-11e6-9622-808a79d2a49b.png)
Also even tried to get my funds out of this wallet via the Copay Recovery Tool BUT that failed as well... scanned with seed words and ended up showing only ZERO BALANCE.

First time I've seen this happen: pretty bad FAIL though, in over a year of using Copay every day. Please fix asap...

P.S. {Edit, update} -- Okay it's working again now, just sent out transaction as usual.  Very odd though, again never saw anything happen like this before: what was the reason?

**Comment by gabegattis** [comment]
BADREQUEST doesn't give us a lot of info. It is a generic error message from BWS. https://github.com/bitpay/bitcore-wallet-service/blob/7f928b0afe92297e3da6559deb4b0da85f6382ab/lib/errors/clienterror.js

What version and OS are you using?

If it happens again, try checking your logs for any error messages. settings->about copay->session log. Errors are in red.

**Comment by KeyJockey** [comment]
Well the only thing RED in current session log text (on Mac OS X version) says: "Cannot verify the format of latest release tag: bitpay" with immediately before that, not in red, "Updating Status: GDAX Wallet null" (GDAX Wallet is name). 

I tried it on three different platforms: my MacBook OS X, my iPhone (older version Copay), and one more instance of the same wallet on Chrome extension running on an old WinXP laptop.  The problem was present in identical form on all three, until it finally resolved and I was able to send the transaction from the Mac OS X version.

If it happens again I'll try and preserve error log better... all wallets were restarted several times since the problem resolved so I assume the real error messages have long since scrolled away?
